proc/proc.go

```diff
@@ -46,6 +46,7 @@ type Process struct {
 	packageMap map[string]string
 
 	allGCache               []*G
+	dwarfLock               sync.Mutex
```

Comment:
We create readers in multiple places in proc and the reader itself isn't thread safe. How do you want to do this?